waddrmgr/db.go

```diff
@@ -1334,14 +1663,12 @@ func upgradeToVersion2(namespace walletdb.Namespace) error {
 
 // upgradeManager upgrades the data in the provided manager namespace to newer
 // versions as neeeded.
-func upgradeManager(namespace walletdb.Namespace) error {
-	// Get the current version.
+func upgradeManager(namespace walletdb.Namespace, pubPassPhrase []byte, obtainSeed, obtainPrivPassPhrase obtainFunc) error {
```

Comment:
As noted below, I suggest changing the signature to accept the entire config struct so it won't have to be changed in the future if any new information is needed.